docs/openstack/logging-in/access-the-openstack-dashboard.md

```diff
@@ -45,4 +45,14 @@ see usage summary for provided date range.
 
 ![OpenStack Horizon dashboard](images/horizon_dashboard.png)
 
+!!! info "I can't find my virtual machine"
+    If you are a member of several projects i.e. ColdFront NERC (OpenStack)
+    allocations, you may need to switch the project before you can see and use the
+    OpenStack resources you or your team has created. Clicking on the project dropdown
+    which is displayed near the top right side will popup the list of projects you
+    are in. You can select the new project by hovering and clicking on the project
+    name in that list as shown below:
+
+    ![OpenStack Project List](images/openstack_project_list.png)
```

Comment:
This image does not desplay for some reason. Maybe the tab?

Reply:
For info box - we can't use tab. This image is also loading slowly.

Reply:
Notice it is now displaying the image but it is displaying text.
![image](https://github.com/nerc-project/nerc-docs/assets/28782106/500bfcf1-9d5f-46ff-960f-8c1f0f38f976)

Reply:
Thank you for showing me on your screen how the !!! info displays differently than Markdown, that helps.